entc/gen/template/builder/create.tmpl

```diff
@@ -61,7 +61,7 @@ func ({{ $receiver }} *{{ $builder }}) Save(ctx context.Context) (*{{ $.Name }},
 			{{- else }}
 				if len({{ $mutation }}.{{ $e.StructField }}IDs()) == 0 {
 			{{- end }}
-				return nil, errors.New("{{ $pkg }}: missing required edge \"{{ $e.Name }}\"")
+				return nil, &ValidationError{Field: "{{ $e.Name }}", err: errors.New("{{ $pkg }}: missing required edge \"{{ $e.Name }}\"")}
```

Comment:
Field?

Reply:
Apart from the discussion below wrgt the reasoning behind this field, would "Label" make more sense to not confuse it between fields/edges?